docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,258 @@
+
+You should see logging output while the cell is running. After execution finishes, you should see output similar to the following:
+
+```bash
+...
+[08/09/22 11:23:30] INFO     Model has accuracy of 0.960 on test data.                                        nodes.py:74
+                    INFO     Saving data to 'metrics' (MetricsDataSet)...                             data_catalog.py:382
+                    INFO     Completed 3 out of 3 tasks                                           sequential_runner.py:85
+                    INFO     Pipeline execution completed successfully.                                      runner.py:89
+```
+
+## Modify your project and test the changes
+
+Now that your project has run successfully once, you can make changes using the convenience and power of your local development environment. In this section, you will modify the project to use a different ratio of training data to test data and check the effect of this change on Databricks.
+
+### Modify the training / test split ratio
+
+The PySpark Iris starter uses a default 80-20 ratio of training data to test data when training the classifier. In this section, you will change this ratio to 70-30 by editing your project in your local environment, then sync it with the Databricks repo using `dbx`, and then run the modified project on Databricks to observe the different result.
+
+Open the file `<project_root>/conf/base/parameters.yml` in your local environment. Edit the line `train_fraction: 0.8` to `train_fraction: 0.7` and save your changes. Look in the terminal where `dbx sync` is running, you should see it automatically sync your changes with your Databricks repo:
+
+```bash
+...
+[dbx][2023-04-14 18:29:39.235] Putting /Repos/<databricks_username>/iris-databricks/conf/base/parameters.yml
+[dbx][2023-04-14 18:29:40.820] Done
+```
+
+### Re-run your project
+
+Return to your Databricks notebook. Re-run the third and fourth cells in your notebook (containing the code `%reload_kedro ...` and `session.run()`). The project will now run again, producing output similar to the following:
+
+```bash
+...
+[08/09/22 11:23:30] INFO     Model has accuracy of 0.953 on test data.                                        nodes.py:74
+                    INFO     Saving data to 'metrics' (MetricsDataSet)...                             data_catalog.py:382
+                    INFO     Completed 3 out of 3 tasks                                           sequential_runner.py:85
+                    INFO     Pipeline execution completed successfully.                                      runner.py:89
+```
+
+You can see that your model's accuracy has changed now that you are using a different classifier to produce the result.
+
+```{note}
+If your cluster terminates, you must re-run your entire notebook, as libraries installed using `%pip install ...` are ephemeral. If not, repeating this step is only necessary if your project's requirements change.
+```
+
+## Summary
+
```

Comment:
I really like this summary but I actually think it would be more useful to have it at the top of the page as a mini-table of contents. Then you could link to the various sections that cover each of the elements.

I'd suggest you put it right at the top before "Prerequisites" and:
* Change the numbered bullets to symbols (because the actual text sections aren't numbered so it'll look a bit odd otherwise)
* Change the gerunds to imperative tense "Create a virtual environment..." , "Create a new Kedro project" etc. etc.

Reply:
Excellent, I really like this suggestion too. Cheers! Made these changes in [bd10a7d](https://github.com/kedro-org/kedro/pull/2482/commits/bd10a7d449841b8aa84ae48fdb216db3e7bb39b1).